src/i18n/strings/en_EN.json

```diff
@@ -1509,15 +1510,16 @@
     "<b>Print it</b> and store it somewhere safe": "<b>Print it</b> and store it somewhere safe",
     "<b>Save it</b> on a USB key or backup drive": "<b>Save it</b> on a USB key or backup drive",
     "<b>Copy it</b> to your personal cloud storage": "<b>Copy it</b> to your personal cloud storage",
-    "Your encryption keys are now being backed up in the background to your Homeserver. The initial backup could take several minutes. You can view key backup upload progress in Settings.": "Your encryption keys are now being backed up in the background to your Homeserver. The initial backup could take several minutes. You can view key backup upload progress in Settings.",
+    "Your keys are being backed up (the first backup could take a few minutes).": "Your keys are being backed up (the first backup could take a few minutes).",
+    "Okay": "Okay",
```

Comment:
fwiw this is somewhat indicative of an inconsistency somewhere - we almost certainly have an `OK` somewhere, but we're using a different one here. Probably not worth fixing at this point though.

Reply:
Bleaugh, you're probably right. One to stomp out on another day, though (unless it's easy to refer to the translation files to identify whether we should Okay or OK)